kvmagent/kvmagent/plugins/storage_device.py

```
@@ -1692,4 +1674,18 @@ def get_nvme_device_info(self, normal_devices):
                 result.append(d)
        
-        return result
\ No newline at end of file
+        return result
+
+    @bash.in_bash
+    def get_megaraid_device_via_device_number_and_bus_number(self, device_number, bus_number):
+        # type: (int, int) -> RaidPhysicalDriveStruct
+        r, vd_info, e = bash.bash_roe("/opt/MegaRAID/storcli/storcli64 /call/vall show all J")
+        if r != 0:
+            raise Exception("can not execute storcli64: %s, stdout: %s, stderr: %s" % (r, vd_info, e))
```

Ruff (0.14.8)

Create your own exception

(TRY002)